src/Map.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import "@opensystemslab/map";
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace JSX {
+    interface IntrinsicElements {
+      "my-map": MapProps;
+    }
+
+    interface MapProps {
```

Comment:
`staticMode` will change this to a "read only" view and remove the zoom buttons.

Docs: https://oslmap.netlify.app/#!/components/vanilla/mymap---basic